docs/extensions/toc.md

```diff
@@ -202,6 +202,8 @@ The following options are provided to configure the output:
 
     The callable must return a string appropriate for use in HTML `id` attributes.
 
+    An alternate version of the default callable supporting unicode strings is also provided as `slugify_unicode`.
```

Comment:
Notice how the `slugify` function is given with its full path above: `markdown.extensions.headerid.slugify`. Let's do the same here: `markdown.extensions.headerid.slugify_unicode`. We don't want to introduce any confusion for our users.